spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerProperties.java

```diff
@@ -36,6 +37,14 @@
 @ConfigurationProperties(prefix = LocalDeployerProperties.PREFIX)
 public class LocalDeployerProperties {
 
+	/**
+	 * Allows logging to be redirected to the output stream of the process that
+	 * triggered child process.
+	 * Could be set per the entire deployment (i.e., <i>deployer.*.inheritLogging=true</i>) or per
+	 * individual application (i.e., <i>deployer.&lt;app-name&gt;.inheritLogging=true</i>).
+	 */
+	public static String INHERIT_LOGGING = AppDeployer.PREFIX + "inheritLogging";
```

Comment:
Also, realizing the prefix should not be `AppDeployer.PREFIX` (ie `spring.cloud.deployer`) but `LocalDeployerProperties.PREFIX` (ie `spring.cloud.deployer.local`)